### DISCUSSION TITLE
Extend a layout/compents

### DISCUSSION THREAD

**Discussion by mohamad-zbib** [opcodesio/log-viewer#194] [Ideas]
In order to merge the page with our system in a good way, we need to extend a layout.

this feature is used in many packages like `livewire`

we can use it in config file like:

    'layout' => 'layouts.admin',

**Comment by arukompas** [comment]
Hey @mohamad-zbib ,

Could you submit a PR to the `1.x` branch? Please avoid the new `2.x` branch because starting from v2.0 the Log Viewer will move away from Livewire and use Vue instead.

If I were you, I'd wait for the new v2 release which will have a few improvements (mobile-friendly, API access), but it won't have the ability to change the layout.
